tests/changelog.spec.js

```diff
@@ -0,0 +1,54 @@
+import { test, expect } from "@playwright/test";
+const AxeBuilder = require("@axe-core/playwright").default;
+
+// Changelog Page
+test("Changelog has title", async ({ page }) => {
+  await page.goto("/changelog");
+  await expect(page).toHaveTitle("LinkFree user changelog");
+});
+
+// Navigating to Changelog Page
+test("Navigate to Changelog", async ({ page }) => {
+  await page.goto("/roadmap/changelog");
```

Comment:
This url does not exist?

This test is for checking that the user can navigate to the page

The test should:
1. go to the `/roadmap` page
2. click on the `changelog` link
3. check the changelog page is displayed

Reply:
okay okay. I am extremely sorry will check everything at once.

Reply:
I have done some changes. Maybe this is okay now.